shared/js/l10n.js

```diff
@@ -2138,4 +2138,29 @@
     whenInteractive(init.bind(navigator.mozL10n, forcePretranslate));
   }
 
+  document.l10n = {
+    setAttributes: navigator.mozL10n.setAttributes,
+    getAttributes: navigator.mozL10n.getAttributes,
+    formatValue: navigator.mozL10n.formatValue.bind(navigator.mozL10n),
+    translateFragment: (...args) =>
+      Promise.resolve(navigator.mozL10n.translateFragment(...args)),
+    ready: () => {
+      return new Promise(function(resolve, reject) {
+        navigator.mozL10n.once(resolve);
```

Comment:
`mozL10n.once` means: context is ready to use.  `l10n.ready` means: the DOM has been localized.  Should we take this difference into account here?